dashboard/views/data_group.py

```diff
@@ -139,6 +149,25 @@ def data_group_detail(request, pk, template_name="data_group/datagroup_detail.ht
         else:
             context["bulkassignprod_form"] = BulkAssignProdForm(dg)
 
+    if dg.include_product_upload_form():
+        if "products-submit" in request.POST:
+            form_data = {
+                "products-TOTAL_FORMS": 0,
+                "products-INITIAL_FORMS": 0,
+                "products-MAX_NUM_FORMS": "",
+            }
+            product_formset = ProductBulkCSVFormSet(form_data, request.FILES)
```

Comment:
```suggestion
            product_formset = ProductBulkCSVFormSet(request.POST, request.FILES)
```
The whole `request.POST` should be passed in. The relevant data will be found via `ProductBulkCSVFormSet.prefix`.

Reply:
Oh, this explains why I could not get the management form data into the formset!

Reply:
I had to roll this change back before it would work again.